soap-stub.js

```diff
@@ -1,5 +1,3 @@
-var _ = require('lodash');
```

Comment:
can we remove lodash altogether?

Reply:
It's just one module. I will try to cover other cases in upcoming PRs.